passenger_car_dynamic_model/include/passenger_car_dynamic_model/PassengerCarDynamicModel.h

```diff
@@ -0,0 +1,140 @@
+#pragma once
+/*
+ * Copyright (C) 2018-2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <vector>
+#include <string>
+#include <memory>
+#include <stdexcept>
+#include <lib_vehicle_model/ODESolver.h>
+#include <lib_vehicle_model/VehicleState.h>
+#include <lib_vehicle_model/VehicleMotionModel.h>
+#include <lib_vehicle_model/VehicleControlInput.h>
+#include <lib_vehicle_model/ParameterServer.h>
+ 
+/**
+ * @class PassengerCarDynamicModel
+ * @brief Class which implements a dynamic vehicle model for front wheel drive passenger cars
+ * 
+ * The details of the math in this class can be found in the PassengerCarDynamicBicycleModel design document
+ * 
+ * NOTE: This class does not support trailers at this time. The trailer angle will remain unchanged during integration
+ * 
+ */
+class PassengerCarDynamicModel: public lib_vehicle_model::VehicleMotionModel
+{
+  private:
+
+    const size_t ODE_STATE_SIZE = 9; // Number of elements of VehicleState that are accounted for in this model
+    const size_t FULL_STATE_SIZE = 12; // Total number of elements in a CARMA VehicleState 
+
+    // Handles to callback functions
+    lib_vehicle_model::ODESolver::ODEFunction<lib_vehicle_model::VehicleControlInput> ode_func_;
```

Comment:
This is because you cannot create function pointers to member variables without a call to std::bind
If the functions were not part of an object they could be passed directly. 

std::function contains conversions for
static function pointers
lamdas
callable objects such as member functions which have been bound